src/core/include/openvino/core/attribute_adapter.hpp

```diff
@@ -89,18 +89,18 @@ class DirectValueAccessor : public ValueAccessor<AT> {
     AT& m_ref;
 };
 
-extern template class DirectValueAccessor<std::vector<std::string>>;
-extern template class DirectValueAccessor<std::vector<signed char>>;
-extern template class DirectValueAccessor<std::vector<unsigned short>>;
-extern template class DirectValueAccessor<std::vector<unsigned int>>;
-extern template class DirectValueAccessor<std::vector<unsigned char>>;
-extern template class DirectValueAccessor<std::vector<short>>;
-extern template class DirectValueAccessor<std::vector<double>>;
-extern template class DirectValueAccessor<std::vector<float>>;
-extern template class DirectValueAccessor<std::vector<int>>;
-extern template class DirectValueAccessor<std::vector<long>>;
-extern template class DirectValueAccessor<std::vector<unsigned long>>;
-extern template class DirectValueAccessor<std::set<std::string>>;
+extern template class OPENVINO_API DirectValueAccessor<std::vector<std::string>>;
```

Comment:
```suggestion
template class OPENVINO_API DirectValueAccessor<std::vector<std::string>>;
```
is `extern` still required here?

Reply:
yes `extern` is required. Otherwise all templates will be explicitly instantiated on every include of attribute_adapter.hpp